src/components/Quote.js

```diff
@@ -1,14 +1,69 @@
-/* eslint-disable linebreak-style */
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
 
 export default function Quote() {
-  return (
-    <section className="quote">
-      <p>
-        Mathematics is not about numbers, equations, computations, or
-        algorithms: it is about understanding.
+  const [quotes, setQuotes] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [hasError, setHasError] = React.useState(false);
+
+  const apiKey = 'QJU4yxMgy1FD96wF65IqoQ==Z4VsyCU2v0Sg2qxc';
+
+  React.useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+          mode: 'cors',
+          method: 'GET',
+          headers: {
+            'X-Api-Key': apiKey,
+            'Content-Type': 'application/json',
+          },
+        });
+        const json = await res.json();
+        setQuotes(json);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setQuotes, setIsLoading, setHasError]);
+
+  const displayQuote = quotes.map((quote) => (
+    <>
+      <p key={1}>
+        {quote.quote}
       </p>
-      <p>- William Paul Thurston</p>
-    </section>
+      <p key={2}>
+        -
+        {' '}
```

Comment:
- As per the project requirements [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Import%20the%20new,as%20the%20Calculator), kindly make sure the quote is on the same page with the calculator.

Your design:
![Screenshot from 2023-03-24 12-14-12](https://user-images.githubusercontent.com/97834160/227494116-78afe781-4eca-4c4a-9c70-223340d58806.png)

Reply:
![image](https://user-images.githubusercontent.com/102612396/227502971-91792def-6d62-4c35-9c91-1dac0893b8f9.png)

@GedeonTS 
I appreciate your efforts, sir, but, there is indeed a place for the quotes as required besides the calculator.

Also, in addition, the requirements indicate that there must be also a separate page for the quote.
Please refer to the sneak peek of the project below. 
https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md